aten/src/ATen/native/Overhead.cpp

```diff
@@ -0,0 +1,29 @@
+#include <ATen/ATen.h>
+
+namespace at {
+
+namespace {
+Tensor trivial_empty() {
+  return detail::empty_generic(
+    /*size=*/{1},
+    /*allocator=*/getCPUAllocator(),
+    /*dispatch_key=*/DispatchKey::CPU,
+    /*scalar_type=*/ScalarType::Float,
+    /*device=*/c10::Device(c10::kCPU),
+    /*memory_format_opt=*/c10::MemoryFormat::Contiguous
+  );
+}
+}  // namespace
+
+namespace native {
+
+Tensor _noop_unary(const Tensor& self) {
+  return trivial_empty();
+}
+
+Tensor _noop_binary(const Tensor& self, const Tensor& other) {
+  return trivial_empty();
+}
```

Comment:
FWIW, in a previous version you had more overloads, and having a ton of tensor inputs (or a tensor list input) is a pretty valid no-op benchmark, since there are some things we do that have overhead that scale with the number of tensors involved.

Reply:
The earlier overloads were for manual kernels, which I removed based on PR feedback. I do like the idea of a tensor list case.